### PR TITLE
GCI-<Devanshk09>: <Golang2>

### PR DESCRIPTION
**Pull request by Devanshk9**

### CODE REVIEW

File: Struct.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    illness        VARCHAR(200),
+    birth_date     DATE,
+    location_id    INT NOT NULL,
+    FOREIGN KEY (location_id) REFERENCES location (id)
+);`
+//structures
+type hospital struct {
+	HospitalID       int    `db:"id" json:"id"`
+	Name             string `db:"name" json:"name"`
+	MaxPatientAmount int    `db:"max_patient_amount" json:"Max patient amount"`
```
Comment (f4ww4z) [inline Struct.go]: The json representation should be `camelCase`. Do this for all the attributes for each model.
```suggestion
	MaxPatientAmount int    `db:"max_patient_amount" json:"maxPatientAmount"`
```

File: main.go [hunk 2]
```diff
@@ -1,28 +1,61 @@
...
+func initDatabase() {
+	var err error
+	db, err = sqlx.Open("postgres", "postgres://tkyandzq:qvN7u6mF6FH8FXspKaSi0rK9AiVEL-wy@john.db.elephantsql.com:5432/tkyandzq")
+	if err != nil {
+		panic(err)
+	}
+}
+
+func GetAllHospitals(c *gin.Context) {
+	var Hospital []hospital
```
Comment (f4ww4z) [inline main.go]: Variable names should be lowercase

File: main.go [hunk 3]
```diff
@@ -1,28 +1,61 @@
...
+	error := db.Select(&Hospital, "SELECT * from Hospital")
+	if error != nil {
+		panic(error)
+	}
+	c.JSON(200, Hospital)
```
Comment (f4ww4z) [inline main.go]: Use `http.StatusOk` instead of `200`

File: main.go [hunk 4]
```diff
@@ -1,28 +1,61 @@
...
+}
+func GetAllLocations(c *gin.Context) {
+	var Location []location
+	error := db.Select(&Location, "SELECT * from Location")
+	if error != nil {
+		panic(error)
+	}
+	c.JSON(200, Location)
```
Comment (f4ww4z) [inline main.go]: Use `http.StatusOk` instead of `200`

File: main.go [hunk 5]
```diff
@@ -1,28 +1,61 @@
...
-		})
-	})
-	//server will run on localhost:8080
+func GetAllPatients(c *gin.Context) {
+	var Patient []patient
+	error := db.Select(&Patient, "SELECT * from Patient")
+	if error != nil {
+		panic(error)
+	}
+	c.JSON(200, Patient)
```
Comment (f4ww4z) [inline main.go]: Use `http.StatusOk` instead of `200`

File: main.go [hunk 6]
```diff
@@ -1,28 +1,61 @@
...
+func GetAllHospitals(c *gin.Context) {
+	var Hospital []hospital
+	error := db.Select(&Hospital, "SELECT * from Hospital")
+	if error != nil {
+		panic(error)
+	}
+	c.JSON(200, Hospital)
+}
+func GetAllLocations(c *gin.Context) {
+	var Location []location
```
Comment (f4ww4z) [inline main.go]: Variable names should be lowercase

File: main.go [hunk 7]
```diff
@@ -1,28 +1,61 @@
...
-	//This create a JSON  /greet/:name endpoint which will greet the person in Hello (name) format where name is a parameter.
-	router.GET("/greet/:name", func(c *gin.Context) {
-		name := c.Param("name")
-		c.JSON(http.StatusOK, gin.H{
-			"message": "Hello " + name,
-		})
-	})
-	//server will run on localhost:8080
+func GetAllPatients(c *gin.Context) {
+	var Patient []patient
```
Comment (f4ww4z) [inline main.go]: Variable names should be lowercase

File: scripts/gci2019_openmrs_go_script.sql [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+CREATE TABLE patient
+(
+    id             SERIAL PRIMARY KEY,
+    name           VARCHAR(100),
+    illness        VARCHAR(200),
+    birth_date     DATE,
+    location_id    INT NOT NULL,
+    FOREIGN KEY (location_id) REFERENCES location (id)
+);
```
Comment (f4ww4z) [inline scripts/gci2019_openmrs_go_script.sql]: Where are your `INSERT` operations?

File: main.go (hunk 2 above)
Comment (Devanshk9) [reply main.go]: done

File: main.go (hunk 7 above)
Comment (Devanshk9) [reply main.go]: Done

File: main.go (hunk 6 above)
Comment (Devanshk9) [reply main.go]: done

File: scripts/gci2019_openmrs_go_script.sql (hunk 8 above)
Comment (Devanshk9) [reply scripts/gci2019_openmrs_go_script.sql]: I actually forget thanks

**Pull request merged**